WordPressKit/WordPressKitTests/PluginServiceRemoteTests.swift

```diff
@@ -34,8 +34,8 @@ class PluginServiceRemoteTests: RemoteTestCase, RESTTestable {
         stubRemoteResponse(sitePluginsEndpoint,
                            filename: getPluginsSuccessMockFilename,
                            contentType: .ApplicationJSON)
-        remote.getPlugins(siteID: siteID, success: { (plugins) in
-            XCTAssertEqual(plugins.count, 8)
+        remote.getPlugins(siteID: siteID, success: { (plugins, _) in
```

Comment:
Just to make this test more complete I would change it to:
```
remote.getPlugins(siteID: siteID, success: { (plugins, sitePluginCapabilities) in
     XCTAssertEqual(plugins.count, 5)
     XCTAssertNotNil(sitePluginCapabilities)
     expect.fulfill()
 }
```
What do you think?

Reply:
Yes, definitely. Just because I'm not using the capabilities in the UI yet it doesn't mean they don't get tested 🤦‍♂️